pkg/cmd/build/build.go

```diff
@@ -246,6 +246,33 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 		file = extractFromContextAndDockerfile(b.Context, b.Dockerfile, svcName)
 	}
 
+	defaultBuildArgs := map[string]string{
```

Comment:
Could we only introduce these default args when the image is being built in okteto?
```suggestion
	if registry.IsOktetoRegistry(b.Image)
```
I don't think we should add those arguments if the image is not being build in the okteto registry

Reply:
done.